lib/builders/dataTypes/multipleTextStringDataTypeBuilder.ts

```diff
@@ -0,0 +1,35 @@
+﻿import {DataTypeBuilder} from "./dataTypeBuilder";
+
+export class MultipleTextStringDataTypeBuilder extends DataTypeBuilder {
+  min: number;
+  max: number;
+
+  constructor() {
+    super();
+    this.editorAlias = "Umbraco.MultipleTextstring";
+    this.editorUiAlias = "Umb.PropertyEditorUi.MultipleTextString";
```

Comment:
Might be a "bug" from frontend that the editorAlias has 'Textstring' while the editorUiAlias is 'TextString